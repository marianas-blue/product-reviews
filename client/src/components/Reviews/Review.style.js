import styled from 'styled-components';

export const Boxes = styled.div`
flex-direction: column;
align-items: stretch;
justify-content: stretch;
padding: 10px;
`;

export const Left = styled.div`
padding-left: 40px;
`;
