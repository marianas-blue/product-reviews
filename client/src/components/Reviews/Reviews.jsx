/* eslint-disable no-useless-constructor */
/* eslint-disable react/prefer-stateless-function */
/* eslint-disable import/extensions */
import React from 'react';
import Images from './Images/Images.jsx';
import Mention from './Mention/Mention.jsx';
import ReviewList from './ReviewList/ReviewList.jsx';
import WriteReview from './WriteReview/WriteReview.jsx';
import { Left, Boxes } from './Review.style';

class Reviews extends React.Component {
  constructor(props) {
    super(props);
  }

  render() {
    return (
      <Left>
        <Boxes><Images /></Boxes>
        <Boxes><Mention /></Boxes>
        <Boxes><ReviewList /></Boxes>
        <Boxes><WriteReview /></Boxes>
      </Left>
    );
  }
}

export default Reviews;
