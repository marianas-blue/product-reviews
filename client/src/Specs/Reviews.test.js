import React from 'react';
import { shallow } from 'enzyme';

import Reviews from '../components/Reviews/Reviews';

describe('Reviews', () => {
  it('should render correctly', () => {
    
    const component = shallow(<Reviews/>);

    expect(component).toMatchSnapshot();
  });
});
