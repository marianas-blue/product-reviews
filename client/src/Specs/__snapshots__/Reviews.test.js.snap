// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Reviews should render correctly 1`] = `
<styled.div>
  <styled.div>
    <Images />
  </styled.div>
  <styled.div>
    <Mention />
  </styled.div>
  <styled.div>
    <ReviewList />
  </styled.div>
  <styled.div>
    <WriteReview />
  </styled.div>
</styled.div>
`;
